admin/test/features/support/sql_db_helper.rb

```diff
@@ -0,0 +1,44 @@
+class SqlDbHelper
+
+  def self.pupil_details(upn)
+    sql = "SELECT * FROM [mtc_admin].[pupil] WHERE upn='#{upn}'"
+    result = SQL_CLIENT.execute(sql)
+    result.first
+  end
+
+  def self.pupil_details_using_names(firstname, lastname)
+    sql = "SELECT * FROM [mtc_admin].[pupil] WHERE foreName='#{firstname}' AND lastName='#{lastname}'"
+    result = SQL_CLIENT.execute(sql)
```

Comment:
its highly likely that users will have same forename and surname.  maybe return set?

Reply:
We will refactor it when we will start replacing the database method in our page objets.
I think we need this method for the newly created pupil which have unique name